From cdba200ab2e4314bb9f200724ad72e85f7179550 Mon Sep 17 00:00:00 2001
From: Dmitriy Aleksandrov <goodmobiledevices@gmail.com>
Date: Mon, 10 May 2021 11:38:19 +0300
Subject: [PATCH] board: nvidia: b3450-0000: setup correct cboot dtb address
 for setup ethernet mac address.

* At startup u-boot get the info from cboot about which address its DTB is in, but it transmits the Kernel DTB address, which does not contain information about the ethernet mac address. Setup address manualy.

Signed-off-by: Dmitriy Aleksandrov <goodmobiledevices@gmail.com>
---
 board/nvidia/p3450-0000/p3450-0000.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/board/nvidia/p3450-0000/p3450-0000.c b/board/nvidia/p3450-0000/p3450-0000.c
index e6b66966..c00f2d94 100644
--- a/board/nvidia/p3450-0000/p3450-0000.c
+++ b/board/nvidia/p3450-0000/p3450-0000.c
@@ -84,7 +84,7 @@ int tegra_pcie_board_init(void)
 
 static void ft_mac_address_setup(void *fdt)
 {
-	const void *cboot_fdt = (const void *)cboot_boot_x0;
+	const void *cboot_fdt = (const void *)0x83000000;
 	uint8_t mac[ETH_ALEN], local_mac[ETH_ALEN];
 	const char *path;
 	int offset, err;
-- 
2.25.1

